casper/src/main/scala/coop/rchain/casper/PrettyPrinter.scala

```diff
@@ -44,6 +44,20 @@ object PrettyPrinter {
       str
     }
 
+  def buildString(d: ProcessedDeploy): String = {
+    val deployString = for {
+      deployData <- d.deploy
+      user       = deployData.user
+      pCost      <- d.cost
+      cost       = pCost.cost
+    } yield s"User: ${buildStringNoLimit(user)}, Cost: ${cost.toString} " +
+            s"${buildString(deployData)}"
+    deployString match {
```

Comment:
You could rewrite this to: `deployString.getOrElse("No deploy data")`.

Reply:
Good one